src/app/tests/suites/certification/Test_TC_PRS_2_1.yaml

```diff
@@ -34,199 +34,111 @@ tests:
-      command: "writeAttribute"
-      attribute: "MinMeasuredValue"
-      arguments:
-          value: 0
-      response:
-          error: UNSUPPORTED_WRITE
-
-    - label: "Reads back mandatory attribute: MinMeasuredValue"
-      command: "readAttribute"
-      attribute: "MinMeasuredValue"
-      response:
-          value: 0
+              minValue: -32768
+              maxValue: 32767
 
     - label: "Read the mandatory attribute constraints: MaxMeasuredValue"
       command: "readAttribute"
       attribute: "MaxMeasuredValue"
       response:
           constraints:
               type: int16
-
-    - label: "Write the default values to mandatory attribute: MaxMeasuredValue"
-      command: "writeAttribute"
-      attribute: "MaxMeasuredValue"
+              minValue: -32768
+              maxValue: 32767
+
+    #Issue 13442 and 15473
+    - label:
+          "Read the optional attribute: Tolerance Verify that the DUT reponds
+          with an uint16 value. Its value should be in the range of 0 and 2048"
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_USER_PROMPT
       arguments:
-          value: 0
-      response:
-          error: UNSUPPORTED_WRITE
-
-    - label: "Reads back mandatory attribute: MaxMeasuredValue"
-      command: "readAttribute"
-      attribute: "MaxMeasuredValue"
-      response:
-          value: 0
-
-    #disabled due to issue #13442
-    - label: "Read the optional attribute: Tolerance"
-      disabled: true
-      command: "readAttribute"
-      attribute: "Tolerance"
-      response:
-          value: 0
-
-    - label: "Write the default values to optional attribute: Tolerance"
-      disabled: true
-      command: "writeAttribute"
-      attribute: "Tolerance"
-      arguments:
-          value: 0
-      response:
-          error: UNSUPPORTED_WRITE
-
-    - label: "Reads back optional attribute: Tolerance"
-      disabled: true
-      command: "readAttribute"
-      attribute: "Tolerance"
-      response:
-          value: 0
-
-    #Following attributes are based on the value of the extended feature in the cluster feature map
-    - label: "Read the optional attribute: ScaledValue"
-      disabled: true
-      command: "readAttribute"
-      attribute: "ScaledValue"
-      response:
-          value: 0
-
-    - label: "Write the default values to optional attribute: ScaledValue"
-      disabled: true
-      command: "writeAttribute"
-      attribute: "ScaledValue"
+          values:
+              - name: "message"
+                value: "Please enter 'y' for success"
+              - name: "expectedValue"
+                value: "y"
+
+    #Issue 13442 and 15473
+    - label:
+          "Read the optional attribute: ScaledValue Verify that the DUT reponds
+          with an int16 value. Its value should be in the range of -32768 and
+          32767"
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_USER_PROMPT
```

Comment:
There again cannot be user prompt: if the chip-tool used by TH does not allow reading via the YAML, the chip-tool that would be used manually also cannot, for the same reason, since it's the same controller code. Please do not convert optional values to user prompts

Reply:
Agree with you there , the reason these were disabled initially was that chip-tool doesn't have support , let me check with the other CSG folks on what the plan is now .Let me circle back with the CSG folks on what the plan was for optional attributes , if we remove it from test plan altogether , its ok but if not we might run into issue of possibly not able to test an incorrect implementation of the optional attribute in SVE.

Reply:
> was that chip-tool doesn't have support

Support for what?

Reply:
Support for these specific optional attributes in SDK

Reply:
Support in chip-tool and YAML is easy: https://github.com/project-chip/connectedhomeip/pull/18189

Support on the server... that depends on what server you are running.

Reply:
@bzbarsky-apple This particular issue we are facing in below mentioned clusters also
1.Window covering
2.Color control
3.Flow measurement
4.Media cluster
5.Occupancy Sensing
6.Power Source
7.Thermostat
8.WiFi Network Diagnostics
let me know if you want me to file issues for this